custom_components/mail_and_packages/const.py

```diff
@@ -13,7 +13,7 @@
 
 DOMAIN = "mail_and_packages"
 DOMAIN_DATA = f"{DOMAIN}_data"
-VERSION = "0.0.0-dev"  # Now updated by release workflow
+VERSION = "0.3.23"
```

Comment:
```suggestion
VERSION = "0.0.0-dev"  # Now updated by release workflow
```
This get's updated when we do a release no need to adjust this.